Jenkinsfile

```diff
@@ -54,12 +54,15 @@ node {
 
     // Deploy
     stage 'Deploy'
-    sh "cat marathon.json | jq \".container.docker.image = \\\"mesosphere/cd-demo-app:${gitCommit()}\\\" | .id = \\\"jenkins-deployed-app\\\" | .labels.lastChangedBy = \\\"${gitEmail()}\\\"\" > marathon-rendered.json"
+
     marathon(
         url: 'http://marathon.mesos:8080',
         forceUpdate: false,
         credentialsId: 'dcos-token',
```

Comment:
The script sets it up automatically for us here: https://github.com/mesosphere/cd-demo/blob/5c01c018c39d594547b7e4e3b28ec4efaa50502b/bin/demo.py#L272. Still worth documenting?